packages/account/package.json

```diff
@@ -36,7 +36,6 @@
         "@deriv/shared": "^1.0.0",
         "@deriv/stores": "^1.0.0",
         "@deriv/translations": "^1.0.0",
-        "@deriv/api": "^1.0.0",
```

Comment:
Is this package not being used anywhere else?

Reply:
This is a duplicate.